### ISSUE TITLE
Disabling `Use animations` setting needs to ensure no animations/transitions are used

### ISSUE THREAD

**Issue by devinbileck** [bisq-network/bisq2#1878]
# Summary

As discovered in #1857, if hardware acceleration is not enabled then the UI may suffer from poor performance.

If not automatically detected, hardware acceleration can be force enabled with `-Dprism.forceGPU=true`. However, this will not be a viable workaround for systems that do not have hardware acceleration such as virtual machines.

As I discovered while investigating #1857, commenting out various animations such as `timeline.play()` improved the performance.

Currently disabling `Use animations` under `Settings > Display settings` does not help. Disabling this needs to ensure no animations/transitions are used at all. Optionally, the app could detect whether hardware acceleration is in use and disable the `Use animations` setting automatically.

# Acceptance Criteria

- All animations/transitions used throughout the UI are inactive when the `Use animations` setting is disabled.

**Comment by HenrikJannsen** [comment]
It should be deactivated everywhere if the use animations flag in the preferences is turned off.
Have you tried that and found still some animations? Might have been forgotten to check at some animations.

**Comment by devinbileck** [comment]
> It should be deactivated everywhere if the use animations flag in the preferences is turned off. Have you tried that and found still some animations? Might have been forgotten to check at some animations.

No, I did not do any further investigation to find specific animations. If someone has some time, would be good to revisit and investigate this.

**Comment by mvp1983** [comment]
I took

**Comment by HenrikJannsen** [comment]
There is a follow up task as described in https://github.com/bisq-network/bisq2/pull/3428
There have been some cases still missing as well.

**Comment by HenrikJannsen** [comment]
I guess that has been resolved in the meantime.
